.changeset/mighty-hairs-train.md

```diff
@@ -0,0 +1,61 @@
+---
+"@comet/admin": minor
+---
+
+Add a more actions menu helper component
+
+This is done to provide a simple api to create a more actions menu for a list of items. 
+
+**Example:**
+
+```tsx
+<MoreActionsMenu
+    selectionSize={selectionSize}
+>
+    {({ handleClose }) => (
+        <>
+            <MenuItem
+                onClick={() => {
+                    handleMoveClick();
+                    handleClose();
+                }}
+            >
+                <ListItemIcon>
+                    <Move />
+                </ListItemIcon>
+                <ListItemText primary={"Move"} />
+            </MenuItem>
+
+            <MoreActionsDivider />
+            
+            <MoreActionsGroup
+                groupTitle={"Some Group Title"}
+            >
```

Comment:
Is this component needed? Shouldn't this be [ListSubheader](https://mui.com/material-ui/api/list-subheader/)?

Reply:
We could us `ListSubheader` but imo the API of the MoreActionsGroup has a better UX. I could abstract the `ListSubheader` and render it in the `MoreActionsGroup` instead of `Typography` and `MenuList` you want 🤷🏻‍♂️